.github/workflows/release_pypi.yml

```diff
@@ -1,17 +1,15 @@
 name: Publish to PyPI
 
 on:
-  # Run on any tag
-  push:
-    tags:
-      - '**'
-  # manually trigger the workflow - for testing only
+  # manually trigger the workflow
```

Comment:
so how do you manually trigger it now? Do you "deploy" the branch?

Reply:
https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow

Like this :)